packages/dashboard/src/components/widgets/tile/tile.tsx

```diff
@@ -58,6 +57,10 @@ const WidgetTile: React.FC<WidgetTileProps> = ({ children, widget, title, remove
   const [visible, setVisible] = useState(false);
   const { iotSiteWiseClient } = useClients();
   const { onDelete } = useDeleteWidgets();
+  const titleRef = useRef<HTMLDivElement | null>(null);
+  const isTitleTruncated =
```

Comment:
is it not possible to solve this with css alone? Otherwise, we can use - `https://github.com/streamich/react-use/blob/master/docs/useMeasure.md` for these types of things.

Reply:
Hi,
I tried to achieve it using 'useMeasure' but it didn't happen because we won't get scrollWidth value from it, to show the tooltip conditionally we need the scrollWidth of the title.
So to avoid this extra code, can I just add a title without a condition, which will display regardless of the long or short title?

Reply:
Hi @jmbuss,

Since 'useMeasure' didn't help to replace the logic I have removed the condition to avoid extra code and just kept the title, I request you to review it once and merge the PR.